### ISSUE TITLE
General Progression Balance

### ISSUE THREAD

**Issue by chloecat34** [MasterEnderman/Zerblands-Remastered#85]
It seems to make an Extreme Crafting Table, for all the Crystal Matrix that goes into it you will need at least 250k Gaia Spirits. Does Woot allow you to make Gaia Spirits, or does the player have to gather them all manually? This seems a bit much to be doable.

Then again the other parts of the Nightmare Mode endgame recipes are basically impossible anyway.

**Comment by MasterEnderman** [comment]
Yes, you can farm the gaia spirits with a Woot farm. I guess I haven't documented that you're able to do so. (Noted for later)

If you have other concerns regarding the balance of certain aspects of the nightmare mode, feel free to report them. In my own playthrough I have not encountered a roadblock I couldn't get passed yet (except the ones I already fixed). Developing and balancing only by yourself is quite a challenge and I appreciate all the feedback I can get.

**Comment by StaffiX** [comment]
> If you have other concerns regarding the balance of certain aspects of the nightmare mode

well I think the draconic -> emc stage wasn't really well tested considering the amount of items required to make a transmutation table

![image](https://github.com/MasterEnderman/Zerblands-Remastered/assets/32968022/c1d0d98c-65f7-4ff9-a19e-32bc9c43516c)
![image](https://github.com/MasterEnderman/Zerblands-Remastered/assets/32968022/4ba5b71a-96c8-4d05-8911-3f0b7e6d347a)
![image](https://github.com/MasterEnderman/Zerblands-Remastered/assets/32968022/cda535a9-3f5c-4449-80c1-2833a44107f9)
![image](https://github.com/MasterEnderman/Zerblands-Remastered/assets/32968022/a67de0bb-7edc-4183-926b-98fe736f5b52)
![image](https://github.com/MasterEnderman/Zerblands-Remastered/assets/32968022/19901998-c1cf-4096-a755-4824ff80734d)
![image)](https://github.com/MasterEnderman/Zerblands-Remastered/assets/32968022/0836dca1-c107-4e42-ab41-362b29aec07a)

well mostly these 4 items that are nothing but waiting for hours, or even years

9 133 000 dragon egg
18 306 000 nether star
408 000 000 draconium (134 040 780 after chaos gurdian + ender dragon)
18 573 000 gaia spirit


> 9 133 000 dragon egg

ender dragon
3000 ticks
->
4 egg
4 heart
32 breath
120 draconium

2 283 250 ender dragon simulation
9 133 000 end crystal
793 hours with 120 woot

---
> 18 306 000 nether star

wither
300 ticks
->
4 star

4 576 500 wither simulation
13 729 500 wither skull
18 306 000 soul sand
158.9 hours with 120 woot

---
> 18 573 000 gaia spirit

(18 138 -> 580 407 -> (for looping))

gaia guardian
3000 ticks
->
32 gaia spirit

580 407 gaia guardian simulation
580 407 gaia ingot
201.5 hours with 120 woot

18 138 gaia simulation
18 138 gaia ingot
6 hours with 120 woot

567 gaia simulation
567 gaia ingot
24 hour with 1 woot

---
but the most brutal waiting might be the amount of mana required to make all the gaiasteel which you need 18 864 112 of it

each gaiasteel is 1.75 mana pool, which means you would need 33 012 196 mana pool

3.3012196e+13 mana
450 mana per second per kekimurus
73360435556 seconds
20377899 hours
2326 years



There is a craftable guilty mana pool that would help with the grind but it requires a whole omega core which needs 9 million! gaia spirit yet again.
I also would suggest maybe adding a custom Terrastrial agglomeration with Multiblocked, and a "mana*" producer with recipes  that uses the flower items, with the actual flowers as catalyst for endgame to have a not laggy way of transfering mana

*where the mana would be an item instead of actual mana.

**Comment by MasterEnderman** [comment]
Thank you for doing all the math and pointing out something that clearly needs some adjustments. In my own playthrough I'm not quite yet at this stage to have encountered this by myself so I appreciate the effort. The easiest would be to rebalance some of there recipes, but I also like you idea of converting botania and it's mana stuff to something that can be fully automated by high speed machines made with multiblocked. That's definitely something I want to add now that you mentioned it. My todo list for the next major update is already quite long and I don't know if I want to prioritize the botania stuff just yet. Time will tell. But thanks again for taking a look at the modpack and giving some much needed feedback. :)
